test/varity/vcf_lift_test.clj

```diff
@@ -0,0 +1,111 @@
+             {:alt ["T"], :ref "TG", :pos 54, :filter [:PASS],
+              :id "2", :info {:END 55}, :qual 100.0, :chr "chr1"}
+             {:alt ["T"], :ref "TG", :pos 54, :filter [:PASS],
+              :id "4", :info {:END 55}, :qual 100.0, :chr "chr1"}
+             {:alt ["GA"], :ref "G", :pos 55, :filter [:PASS],
+              :id "1", :info {:END 55}, :qual 100.0, :chr "chr1"}
+             {:alt ["T"], :ref "G", :pos 55, :filter [:PASS],
+              :id "3", :info {:END 55}, :qual 100.0, :chr "chr1"}]
+            :failure nil}))
+    (is (= (vcf-lift/liftover-variants
+            seq-reader
+            (ch/index
+             (ch/load-chain "./test-resources/vcf-lift/sample-rev.chain"))
+            (vcf/read-variants vcf-reject-rdr))
+           {:failure [{:alt ["A"], :ref "AT", :pos 5, :filter [:PASS], :id nil,
+                       :info {:END 6}, :qual 100.0, :chr "chr1"}]}))))
+
+(deftest vcf-forward-lift-test
+  (with-open [vcf-forward-rdr
+              (vcf/reader "./test-resources/vcf-lift/sample-forward.vcf")
+              seq-reader
+              (io-seq/reader "./test-resources/vcf-lift/sample.fasta")]
+    (is (= (vcf-lift/liftover-variants
+            seq-reader
+            (ch/index (ch/load-chain
+                       "./test-resources/vcf-lift/sample-forward.chain"))
+            (vcf/read-variants vcf-forward-rdr))
+           {:success [{:alt ["AT"], :ref "A", :pos 5, :filter [:PASS],
+                       :id "1", :info {:END 5}, :qual 100.0, :chr "chr1"}
+                      {:alt ["ATA" "G"], :ref "AT", :pos 5, :filter [:PASS],
+                       :id "2", :info nil, :qual 100.0, :chr "chr1"}
+                      {:alt ["CG"], :ref "C", :pos 8, :filter [:PASS],
+                       :id "3", :info {:END 8}, :qual 100.0, :chr "chr1"}]
+            :failure nil}))))
+
+(deftest vcf-multi-lift-test
+  (with-open [vcf-forward-rdr
+              (vcf/reader "./test-resources/vcf-lift/sample-multi.vcf")
+              seq-reader
+              (io-seq/reader "./test-resources/vcf-lift/sample.fasta")]
+    (is (= (vcf-lift/liftover-variants
+            seq-reader
+            (ch/index (ch/load-chain
+                       "./test-resources/vcf-lift/sample-multi.chain"))
+            (vcf/read-variants vcf-forward-rdr)))
+        {:failure nil,
+         :success [{:alt ["A"], :ref "ATGCATGC", :pos 9, :filter [:PASS],
+                    :id "1", :info {:END 16}, :qual 100.0, :chr "chr1"}
+                   {:alt ["A"], :ref "ATGCATGC", :pos 9, :filter [:PASS],
+                    :id "3", :info {:END 16}, :qual 100.0, :chr "chr1"}
+                   {:alt ["A"], :ref "ATGCATGCATGCATGCATGC", :pos 9,
+                    :filter [:PASS], :id "2", :info {:END 28}, :qual 100.0,
+                    :chr "chr1"}
+                   {:alt ["T"], :ref "TG", :pos 54, :filter [:PASS], :id "4",
+                    :info {:END 55}, :qual 100.0, :chr "chr1"}]})))
```

Comment:
```suggestion
    (is (= (vcf-lift/liftover-variants
            seq-reader
            (ch/index (ch/load-chain
                       "./test-resources/vcf-lift/sample-multi.chain"))
            (vcf/read-variants vcf-forward-rdr))
           {:failure nil,
            :success [{:alt ["A"], :ref "ATGCATGC", :pos 9, :filter [:PASS],
                       :id "1", :info {:END 16}, :qual 100.0, :chr "chr1"}
                      {:alt ["A"], :ref "ATGCATGC", :pos 9, :filter [:PASS],
                       :id "3", :info {:END 16}, :qual 100.0, :chr "chr1"}
                      {:alt ["A"], :ref "ATGCATGCATGCATGCATGC", :pos 9,
                       :filter [:PASS], :id "2", :info {:END 28}, :qual 100.0,
                       :chr "chr1"}
                      {:alt ["T"], :ref "TG", :pos 54, :filter [:PASS], :id "4",
                       :info {:END 55}, :qual 100.0, :chr "chr1"}]}))))
```

Reply:
I decided that simple code(https://github.com/chrovis/varity/pull/35#issuecomment-680415666) didn't work before, but it seems that I made a mistake when I checked.

I replaced with the simple code.
The result of throwing the Picard test was no problem.